kedro-datasets/CONTRIBUTING.md

```diff
@@ -23,12 +23,48 @@ If you have already checked the [existing issues](https://github.com/kedro-org/k
 
 If you have new ideas for Kedro-Datasets then please open a [GitHub issue](https://github.com/kedro-org/kedro-plugins/issues) with the label `enhancement`. Please describe in your own words the feature you would like to see, why you need it, and how it should work.
 
-### Contribute a new dataset
+## Contribute a new dataset
 
 If you're unsure where to begin contributing to Kedro-Datasets, please start by looking through the `good first issue` and `help wanted` on [GitHub](https://github.com/kedro-org/kedro-plugins/issues).
 If you want to contribute a new dataset, read the [tutorial to create and contribute a custom dataset](https://docs.kedro.org/en/stable/data/how_to_create_a_custom_dataset.html) in the Kedro documentation.
 Make sure to add the new dataset to `kedro_datasets.rst` so that it shows up in the API documentation and to `static/jsonschema/kedro-catalog-X.json` for IDE validation.
 
+Below is a guide to help you understand the process of contributing a new dataset, whether it falls under the category of regular or experimental datasets.
+
+### Difference between regular and experimental datasets
+
+#### Regular datasets
+Regular datasets are maintained by the [Kedro Technical Steering Committee (TSC)](https://docs.kedro.org/en/stable/contribution/technical_steering_committee.html) and adhere to specific standards. These datasets adhere to the following requirements:
+
+1. Must be something that the Kedro TSC is willing to maintain.
+2. Must be fully documented.
+3. Must have working doctests (unless complex cloud/DB setup required, which can be discussed in the review).
+4. Must run as part of the regular CI/CD jobs.
+5. Must have 100% test coverage.
+6. Should support all Python versions under NEP 29 (3.9+ currently).
+7. Should work on Linux, macOS, and Windows.
+
+#### Experimental datasets
+The requirements for experimental datasets are more flexible and these datasets are not maintained by the Kedro TSC. Experimental datasets:
```

Comment:
Should we provide an example - e.g. a new DataFrame framework which hasn't yet seen mass adoption or a niche Simulation format.

Reply:
I thought about that, but if the TSC is willing to maintain it even something like that could become a "core" contribution.